seed/index.js

```diff
@@ -0,0 +1,55 @@
+// const House = require('../db/Schemas/Listing.js');
+
+
+//loop 100 times
+const seed = () => {
+    const houses = [];
+    for (let i = 0; i < 100; i++) {
+        const house = {};
+        let pad = i;
+
+        if (pad.toString().length < 2) {
+            house['Listing_id'] = '00' + pad.toString();
+        } else {
+            house['Listing_id'] = '0' + pad.toString();
+        }
+
+        house['Price'] = Math.floor(Math.random() * 5999999) + 100000;
+
+        const Detail = {};
+
+        Detail['Room_count'] = Math.floor(Math.random() * 6) + 2;
+        Detail['Bathroom_count'] = Math.floor(Math.random() * 3) + 1;
+        Detail['Square_footage'] = Math.floor(Math.random() * 2600) + 500;;
+
+        house['Details'] = Detail;
+
+        const Address = {};
+        Address['House_number'] = Math.floor(Math.random() * 9888) + 111;
+        Address['Street_number'] = Math.floor(Math.random() * 300);
+
+        const street = ['street', 'avenue', 'place', 'boulevard'];
+        const direction = ['', ' NW', ' NE', ' S', ' SW', ' SE', ' N'];
```

Comment:
No 'E' or 'W' only addresses in Seattle?  Is this a thing I have never noticed?

Reply:
I made it up :)